cmd/security-bootstrapper/entrypoint.sh

```diff
@@ -37,6 +37,7 @@ EDGEX_USER_ID=${EDGEX_USER:-$DEFAULT_EDGEX_USER_ID}
 # which then injecting into all other related containers on other services' entrypoint scripts
 # if the executable is not 'security-bootstrapper'; then we consider it not running the bootstrapping process
 # for the user may just want to debug into the container shell itself
+cp -rpd /edgex-init-staging/* /edgex-init/
```

Comment:
This cp should use the env vars defined in the dockerfile.  Copy from STAGING_DIR to INIT_DIR.